bin/node/runtime/src/lib.rs

```diff
@@ -332,6 +332,7 @@ parameter_types! {
 	pub const ExpectedBlockTime: Moment = MILLISECS_PER_BLOCK;
 	pub const ReportLongevity: u64 =
 		BondingDuration::get() as u64 * SessionsPerEra::get() as u64 * EpochDuration::get();
+	pub const MaxAuthorities: u32 = 100;
```

Comment:
Probably we should document that this should be the upper bound of what `Staking::validator_count` is ever going to be.